pages/inventory.vue

```diff
@@ -45,42 +45,48 @@
                   v-if="i == 1"
                   :headers="headers"
                   :items="inventory.microcontroller"
-                  hide-default-footer
                   class="elevation-1 mb-2"
                 >
                   <template v-slot:items="props">
                     <td>{{ props.item.name }}</td>
```

Comment:
props is redundant. Remove `props.` from everywhere

Reply:
I tried removing the __props variable__ from v-slot but it's not working.

```
Property or method "item" is not defined on the instance but referenced during render. Make sure that this property is reactive, either in the data option, or for class-based components, by initializing the property.
TypeError: Cannot read property 'name' of undefined at inventory.js:339
```